dockerfiles/cfme_tests_dev/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Intended build context is integration_tests top level
+
+FROM sel-browsers:latest
+
+ENV PYCURL_SSL_LIBRARY=nss \
+    PYTHONDONTWRITEBYTECODE=1 \
+    VENV=/cfme_venv \
+    CFME_ENV=/integration_tests
+
+COPY ./ $CFME_ENV
+WORKDIR $CFME_ENV
+# for package installs through quickstart
+USER 0
+RUN python2 -m cfme.scripting.quickstart --mk-virtualenv $VENV
+RUN source $VENV/bin/activate && pip freeze && pip install -e .
+
+# integration_tests volume mounted at run time to pickup real-time changes
+VOLUME $CFME_ENV
+WORKDIR $CFME_ENV
+
+COPY ./dockerfiles/cfme_tests_dev/docker-assets/entrypoint /entrypoint
+
+
+
+EXPOSE 5999
+
+ENTRYPOINT ["/entrypoint"]
```

Comment:
If I were to run this, how would I build and invoke the container? And I see this is `for dev` but what if I want to run against regular appliance? Do we have container for that?

Reply:
Reason I ask this is that, all of us have different way to setup and run the tests, I use selenium container for browser, some use local browser installed on their system, etc. What would be a way to unify that and make it such that all of QE team can standardize on a container image that acts as test env for them, includes right dependencies and browsers. We then can rebuild that as needed and team won't need to fix their env, just pull latest docker image? I'm sure you must have thought about it. Have you?